statviz/src/components/graphs/BarChartCenterAxis.tsx

```diff
@@ -48,14 +48,14 @@ const defaultSettings = {
   hideZeroX: false,
 };
 
-const marginTop = 20;
-const marginLeft = 45;
+const marginTop = 0;
+const marginLeft = 70;
 const marginRight = 40;
-const marginBottom = 40;
+const marginBottom = 70;
 
 const labelProps = {
```

Comment:
in the future, I would love if we could somehow just load it from the customized chakra theme.

Reply:
No ACTION ITEM atm. Just something to consider for the future